tests/test-files/transform/MissingImportParens.elm

```diff
@@ -0,0 +1,14 @@
+module Main exposing (x)
+
+import Import.As1 as ImportAs1
+import Import.As2 as ImportAs2 exposing A
+import ImportDotDot exposing ..
+import ImportName exposing A
+import ImportNames exposing A, B, x, y
+import OtherImport1 exposing (A(X, Y), b, z)
+import OtherImport2 exposing (..)
+import OtherImport3
```

Comment:
The "OtherImport*" lines are testing things that (hopefully!) should be tested in other files (in particular, somewhere in `tests/test-files/good/**`).  I'd probably leave them out here for that reason, though I could be convinced either way on this.

Reply:
You're right. My intention to have the good cases, was not to break them. While developing I had cases with a wrong parser, where the good case failed (because too much was parsed?). I'll reduce the standard cases, but I think it makes sense to have them here, to test the case, that there is a mix of different possibilities, and they they're working together in the same file. Also adding the merge case.